pkg/container-utils/types/types.go

```diff
@@ -16,9 +16,14 @@ package types
 
 import "github.com/inspektor-gadget/inspektor-gadget/pkg/types"
 
+const (
+	RuntimeProtocolInternal = "internal"
+	RuntimeProtocolCRI      = "cri"
+)
+
 type ExtraConfig struct {
-	Namespace string
-	UseCri    bool
+	Namespace       string
+	RuntimeProtocol string
 }
 
 type RuntimeConfig struct {
```

Comment:
IMO, it shouldn't go in `ExtraConfig` but directly in the `RuntimeConfig` and be mandatory (it will be automatically populated in most paths by the default flag value). The `ExtraConfig` struct should remain a pointer with the optional parameters as it was before. WDYT?

Reply:
I updated the last commit with that change.
Note that `ExtraConfig` was not a pointer.

Reply:
Mmmm it was, but Burak modified it in a commit of this PR: 

https://github.com/inspektor-gadget/inspektor-gadget/blob/1072df20edbf07eed8269dd896494e4c1bc0afd8/pkg/container-utils/types/types.go#L23-L27